src/css/protocol/innovation.scss

```diff
@@ -110,3 +111,31 @@ a.mzp-c-button {
         }
     }
 }
+
+.c-navigation {
+    box-shadow: 0 0 1px 1px rgba(0, 0, 0, 0.1) !important; /* stylelint-disable-line no-important */
```

Comment:
There is some specificity trouble with some of the CSS overrides (namely this one but I could forsee it happening elsewhere as well). This is largely due to the order in which various style sheets are declared. Since I think maintaining a strict cascade order may be harder to accomplish, we discussed adding the theme name to the `body` class so we can have an extra style hook when needed for specificity in theme overrides. Once that gets added I can update the CSS and remove this nasty `!imporant`